src/app.js

```diff
@@ -15,14 +15,15 @@ async function run() {
 	let mongoOperator;
 	let amqpOperator;
 	let validator;
-
+	let toMarcRecords;
 	try {
 		mongoOperator = await mongoFactory(MONGO_URI);
 		amqpOperator = await amqpFactory(AMQP_URL);
 		validator = await validatorFactory();
+		toMarcRecords = await toMarcRecordFactory(amqpOperator);
 	} catch (error) {
 		logError(error);
-		process.exit(5);
+		process.exit(0);
 	}
```

Comment:
I don't see the need to wrap these in try-catch and using let. If calling these functions fail the application will exit anyway and no custom error handler is needed.